onediff_diffusers_extensions/examples/flux/text_to_image_flux.py

```
@@ -0,0 +1,407 @@
+        self.begin = None
+        self.end = None
+        self.num_iterations = 0
+
+    def get_iter_per_sec(self):
+        if self.begin is None or self.end is None:
+            return None
+        self.end.synchronize()
+        dur = self.begin.elapsed_time(self.end)
+        return self.num_iterations / dur * 1000.0
+
+    def callback_on_step_end(self, pipe, i, t, callback_kwargs={}):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)